app/templates/components/my-application.hbs

```diff
@@ -1,59 +1,59 @@
-      {{/if}}
-
-      {{#if isRecruiter}}
-        {{#link-to 'recruiter' class="navbar-brand" }}
-          {{#paper-button}}{{paper-icon 'home'}}Home{{/paper-button}}
-        {{/link-to}}
-      {{/if}}
-    {{else}}
-      {{#link-to 'application' class="navbar-brand" }}
-        {{#paper-button}}{{paper-icon 'home'}}Home{{/paper-button}}
-      {{/link-to}}
+    <h1 class="text-center header-title">Free Lunch</h1>
+{{/paper-toolbar}}
+
+<div class="toolbar-two">
+  {{#if isAuthenticated}}
 
+    {{#if isEngineer}}
+      {{#link-to 'engineer' class="navbar-brand" }}{{#paper-button raised=true }}{{paper-icon 'home'}}{{/paper-button}}{{/link-to}}
     {{/if}}
-    <div class="tool-bar-spacer"></div>
-    <h1 class="text-center header-title">Free Lunch</h1>
 
-    <div class="paper-right">
-      {{#if isAuthenticated}}
-        {{#link-to "change-password"}}
-          {{#paper-button}}{{paper-icon 'settings'}}Change Password{{/paper-button}}
-        {{/link-to}}
-        {{#paper-button}}
-          <div {{action "signOut"}}>Sign Out</div>
-        {{/paper-button}}
-      {{else}}
-        {{#link-to "sign-up"}}
-          {{#paper-button}}Sign Up{{/paper-button}}
-        {{/link-to}}
-        {{#link-to "sign-in"}}
-          {{#paper-button}}Sign In{{/paper-button}}
-        {{/link-to}}
-      {{/if}}
-    </div>
-  </div>
+    {{#if isRecruiter}}
+      {{#link-to 'recruiter' class="navbar-brand" }}
+        {{#paper-button raised=true }}{{paper-icon 'home'}}{{/paper-button}}
+      {{/link-to}}
+    {{/if}}
+
+  {{else}}
+    {{#link-to 'application' class="navbar-brand" }}
+      {{#paper-button raised=true}}Home{{/paper-button}}
+    {{/link-to}}
+  {{/if}}
 
-{{/paper-toolbar}}
-<div class="toolbar-two">
   {{#if isAuthenticated}}
 
     {{#if isRecruiter}}
-      {{#link-to 'recruiter.engineers'}}{{#paper-button raised=true}}View Engineers{{/paper-button}}{{/link-to}}
-      {{#link-to 'recruiter.conversations'}}{{#paper-button raised=true}}View Lunches{{/paper-button}}{{/link-to}}
+      {{#link-to 'recruiter.engineers'}}
+        {{#paper-button raised=true}}View Engineers{{/paper-button}}{{/link-to}}
+      {{#link-to 'recruiter.conversations'}}
+        {{#paper-button raised=true}}View Lunches{{/paper-button}}
+      {{/link-to}}
     {{/if}}
 
     {{#if isEngineer}}
-      {{#link-to 'engineer.conversations'}}{{#paper-button raised=true }}View Lunch Invites{{/paper-button}}{{/link-to}}
-
+      {{#link-to 'engineer.conversations'}}
+        {{#paper-button raised=true }}View Lunch Invites{{/paper-button}}
+      {{/link-to}}
     {{/if}}
+
   {{/if}}
 
+  <div class="paper-right">
+    {{#if isAuthenticated}}
+      {{#link-to "change-password"}}
+        {{#paper-button raised=true}}Change Password{{/paper-button}}{{/link-to}}
+      {{#paper-button raised=true}}
+        <div {{action "signOut"}}>Sign Out</div>
+      {{/paper-button}}
+
+    {{else}}
+      {{!-- inconsistent indentation fixes bug with link-to --}}
```

Comment:
I filed an issue on the repo for ember-paper.

https://github.com/miguelcobain/ember-paper/issues/745